labelmerge/workflow/scripts/label_split.py

```diff
@@ -52,9 +73,14 @@ def label_split(atlas_path, output_dir, smk_wildcards):
         )
 
         # Grab label description
-        label_desc = atlas_metadata[atlas_metadata["Index"] == int(label)][
-            "BIDS Name"
-        ].values[0]
+        try:
+            label_desc = atlas_metadata[atlas_metadata["Index"] == int(label)][
+                "BIDS Name"
+            ].values[0]
+        except:
```

Comment:
is there going to be a case where the user has a very detailed map and wants to simply select the ones in their description? (i.e., nii file will have lots of labels, user is interested in ones in their description file)

Reply:
Maybe. But that would still require the original labels to be split. Thought selection would be handled on the merging side (potentially in v0.2.x)

Reply:
Ahh, I think I see what you mean now. The other way to implement this is to loop through the labels provided in the description rather than loop through all of the labels. That said, the two should be equal?

Reply:
Once we've actually implemented the combination schema API we can revisit this